R/derive_date_vars.R

```diff
@@ -134,7 +134,7 @@
 #' # minimum dates
 #' impute_dtc(
 #'   "2020-12",
-#'   min_dates = list(
+#'   min_dates = vars(
 #'     ymd_hms("2020-12-06T12:12:12"),
 #'     ymd_hms("2020-11-11T11:11:11")
 #'   ),
```

Comment:
Only make the change when `derive_vars_dt()` or `derive_vars_dtm()` is called but **not** for `impute_dtc()`.